RELEASE-NOTES.md

```diff
@@ -25,38 +29,28 @@ The binaries are tested on the following platforms:
 The macOS package has a hard dependency on OpenSSL v1.0.0 installed with [Homebrew](https://brew.sh/) in its default path `/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib`.
 In case you have installed it in a non-default path, you could use a symlink to work around the issue.
 
+Alternatively to the release binaries, you can use the published `aetrnty/epoch` Docker image, documented at [this location](https://github.com/aeternity/epoch/blob/v0.5.0/docs/docker-testnet.md).
+
 The user configuration is documented in the [wiki](https://github.com/aeternity/epoch/wiki/User-provided-configuration) though the instructions below contain easy-to-use examples.
 
-HTTP endpoints are documented in the [swagger.yaml](https://github.com/aeternity/epoch/blob/v0.4.1/config/swagger.yaml). A swagger.json version of the same documentation is present in the release: `lib/aehttp-0.1.0/priv/swagger.json`
+HTTP API endpoints are specified in the [swagger.yaml](https://github.com/aeternity/epoch/blob/v0.5.0/config/swagger.yaml); a swagger.json version of the same specification is present in the release binary at path `lib/aehttp-0.1.0/priv/swagger.json`.
+WebSocket API endpoints are specified at [this location](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/epoch_api.md).
 
-There is also more general [Epoch Node API](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/epoch_api.md) documentation.
+The intended usage of the API (HTTP and WebSocket) of the node is documented at [this location](https://github.com/aeternity/protocol/blob/epoch-v0.5.0/epoch_api/epoch_api.md).
 
 ## Join the testnet
 
-This section describes how to run a node as part of the testnet.
+This section describes how to run a node as part of the testnet - the public test network of nodes - by using the release binary.
 
 ### Inspect the testnet
 
-The core nodes of the public test network are accessible from the Internet and expose [an HTTP API](https://github.com/aeternity/epoch/blob/v0.4.1/config/swagger.yaml).
+The core nodes of the public test network are accessible from the Internet.
 
 Information, e.g. height, of the top block of the longest chain as seen by these core nodes of the testnet can be obtained by opening in the browser any of the following URLs:
 * http://31.13.248.103:3013/v2/top
 * http://31.13.248.102:3013/v2/top
 * http://31.13.248.105:3013/v2/top
 
-
-### Using Docker
```

Comment:
@dincho I summarized this section in a line above. Also I avoided the link "docs/docker-testnet.md" as not easy to follow for the user IMO.